src/Monomer/Widgets/Singles/TextArea.hs

```diff
@@ -762,7 +762,7 @@ stateFromText wenv node state text = newState where
   style = currentStyle wenv node
   fontMgr = wenv ^. L.fontManager
   newTextMetrics = getTextMetrics wenv style
-  tmpTextLines = fitTextToWidth fontMgr style maxNumericValue KeepSpaces text
+  tmpTextLines = fitTextToWidth fontMgr style maxNumericValue KeepSpaces OnSpaces text
```

Comment:
It's also fine if we ignore the new text attribute here since it never forces a break unless a `\n` is provided (it passes the largest number available as the line break width)